tests/python/unittest/test_tvmscript_parser_evaluator.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Unittests for tvm.script.parser.evaluator"""
+import pytest
+import tvm.testing
+from tvm.script._parser.core.diagnostics import Source
+from tvm.script._parser.core.evaluator import ExprEvaluator
+
+
+def calc(expr, extra_vars={}):
```

Comment:
1. Start the name with an underscore
2. Use “None” as default for extra_vars

Reply:
Fixed